api/test/h3-data/h3-material-map.spec.ts

```diff
@@ -106,4 +111,31 @@ describe('H3 Data Module (e2e) - Material map', () => {
       unit: 'tonnes',
     });
   });
+
+  test('When I query H3 data at minimal resolution, then I should get 4 h3indexes with expected values and no 0 as value', async () => {
+    const h3Data = await h3AlternativeDataMock(
+      fakeTable,
+      fakeColumn,
+      h3AlternativeFixture,
+    );
+    const material = await createMaterial({ producer: h3Data });
+    const response = await request(app.getHttpServer()).get(
+      `/api/v1/h3/map/material?materialId=${material.id}&resolution=1`,
```

Comment:
look into the .query() to add query arguments, instead of adding them to the URL string.

Reply:
+1